app/src/test/java/nl/tudelft/b_b_w/controller/BlockControllerUnitTest.java

```diff
@@ -402,6 +406,30 @@ public void testBacktrack() {
     }
 
     /**
+     * verifyTrustworthiness test
+     * Tests whether a block is trustworthy
+     */
+    @Test
+    public void testVerifyTrustworthiness() {
+        final Block newBlock = BlockFactory.getBlock(TYPE_BLOCK, owner, bc.getLatestSeqNumber(owner)+1,
```

Comment:
For now this is OK. There is a PR upcoming that will use the new User class for both owner and contact, which will include a refactor of the BlockFactory as well, so this will get fixed there as well.